Adyen/UI/Form/Items/Text/FormTextItemView.swift

```diff
@@ -292,12 +292,14 @@ open class FormTextItemView<ItemType: FormTextItem>: FormValueItemView<String, F
 
     /// :nodoc:
     open func updateValidationStatus(forced: Bool = false) {
-        if item.isValid() {
+        let hasText = textField.text.map(\.isEmpty) == false
```

Comment:
What do you think about renaming this variable to something more descriptive like `isTextFieldEmpty` and rewrite the condition as `force || !isTextFieldEmpty`?

Or keep it like this and just name it `isTextFieldNotEmpty`

Reply:
👍